pkg/bisect/bisect.go

```diff
@@ -72,18 +73,18 @@ const NumTests = 10 // number of tests we do per commit
 //  - if the crash still happens on the oldest release/HEAD (for cause/fix bisection correspondingly),
 //    no commits and the crash report on the oldest release/HEAD
 //  - if the crash is not reproduced on the start commit, an error
-func Run(cfg *Config) ([]*vcs.Commit, *report.Report, error) {
+func Run(cfg *Config) ([]*vcs.Commit, *report.Report, *vcs.Commit, error) {
```

Comment:
Please update the comment above. What/when exactly this function returns is quite tricky and it becomes even more tricky now.